src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -44,7 +55,8 @@ TopicDomainHeader.propTypes = {
   handleOnMemberPress: PropTypes.func,
   hideSeeMember: PropTypes.bool,
   isFollow: PropTypes.bool,
-  memberCount: PropTypes.number
+  memberCount: PropTypes.number,
+  isLoading: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `isLoading` to PropTypes is a good practice for type checking in React. However, it would be better if we also provide a default value for `isLoading` in case it's not provided.
```diff
  TopicDomainHeader.defaultProps = {
    handleOnMemberPress: null,
    hideSeeMember: false,
    isFollow: false,
+   isLoading: false,
    memberCount: 0,
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ah ya good suggestion